### PR TITLE
Added Icon Reference

### LINKED ISSUE #84: My questions symbol key
Adding a key map for symbols on the my questions page.

### PR DESCRIPTION
**Pull request by erinisbell**
Closes #84

### CODE REVIEW

**Comment by tjgragg** [comment]
1) Questions answer question
     Failure/Error: expect(page).to have_content(answer)
       expected to find text "You just put a bird on it! Duh." in "Questions Logout Animi sed saepe repellat facilis provident.Sheila Gaylord DELETEEDITRESOLVEAsked less than a minute ago. note_add Answer SUBMIT"

This is the test that kept failing intermittently before. The submitted code has nothing to do with the test and would be a cause of failure.

File: app/views/dashboards/home.slim [hunk 1]
```diff
@@ -2,6 +2,15 @@
   h1
     ' Welcome
     = current_user.name
+    <br>
+  <h5> Icon Refrence </h5>
```
Comment (kurtisnelson) [inline app/views/dashboards/home.slim]: HTML mixed in with slim

File: app/views/dashboards/home.slim [hunk 2]
```diff
@@ -12,3 +21,5 @@
 - if current_user
   .fixed-action-btn style="bottom: 45px; right: 24px;"
     = link_to material_icon.add, new_question_path, class: "btn-floating btn-large waves-effect waves-light teal"
+    
+= react_component('Answer', {})
```
Comment (kurtisnelson) [inline app/views/dashboards/home.slim]: I don't think this component exists yet.

Comment (erinisbell) [inline app/views/dashboards/home.slim]: This has been deleted from the code.

File: app/views/dashboards/home.slim (hunk 1 above)
Comment (erinisbell) [inline app/views/dashboards/home.slim]: The HTML has been deleted.

**Pull request merged**